sdk/src/lib.rs

```diff
@@ -135,12 +135,18 @@ pub mod axiom {
     };
 }
 /// Contains a CLI for running any Axiom Compute function (any struct that implements the `AxiomComputeFn` trait)
-pub mod cmd;
+pub mod cli;
 /// Contains the traits and types required to implement an Axiom Compute function (re-exported from the `axiom` module)
 pub(crate) mod compute;
+/// Contains a web server for running any Axiom Compute function (any struct that implements the `AxiomComputeFn` trait)
+pub mod server;
 /// Module with all subquery types and builders
 pub mod subquery;
 /// Re-export ethers-rs
 pub use ethers;
+/// Run either the proving/keygen CLI or the server from the same binary
+pub mod bin;
```

Comment:
I think it may be more standard to call this `cli` and then `cli` should be called `run` or `bin`. Usually `cli` refers to the top level thing.